.github/workflows/ci.yaml

```diff
@@ -387,7 +387,7 @@ jobs:
 
   deploy:
     name: "deploy"
-    runs-on: ${{ github.repository_owner == 'coder' && 'buildjet-8vcpu-ubuntu-2204' || 'ubuntu-latest' }}
+    runs-on: ${{ github.repository_owner == 'coder' && 'buildjet-16vcpu-ubuntu-2204' || 'ubuntu-latest' }}
```

Comment:
This seems kinda moot since `deploy` only runs on main and it doesn't matter that much how long it takes.

Reply:
My thinking is if it’s well parallelized then it costs the same to give it more resources.